containers/images/foreman/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install http://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm && \
+    yum -y install https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-repos-latest.rpm && \
+    yum -y install foreman-release-scl && \
+    yum -y install qpid-tools && \
+    yum -y install foreman tfm-rubygem-puma && \
+    yum -y install tfm-rubygem-katello tfm-rubygem-foreman_ansible tfm-rubygem-foreman_discovery tfm-rubygem-foreman_hooks && \
+    yum -y install tfm-rubygem-foreman_memcache tfm-rubygem-foreman_openscap tfm-rubygem-foreman_remote_execution tfm-rubygem-foreman_templates && \
```

Comment:
you can squash multiple yum executions in to a fewer ones, it would make image creation faster. (e.g. )
```
yum -y install qpid-tools foreman tfm-rubygem-puma ....
```
Probably better to expose those as an ARG so people can add additional packages if they rebuild it (with an additional plugin?)

Reply:
Fair, I'll update some of them, but I don't want to lose the readability and the implications that each is a unique step to be done in a particular order. Sometimes that is my gripe with these "ugly" ways you are forced to write Dockerfiles rather than the build tooling being able to optimize.

As for making the package list an ARG, I'd like to translate as much as possible things directly first. To reduce the amount of churn in conversion. Then come back through and add in features.